src/polkadot/PolkadotEngine.ts

```diff
@@ -224,6 +224,16 @@ export class PolkadotEngine extends CurrencyEngine<
   }
 
   async queryTransactions(): Promise<void> {
+    /*
+    HACK: We cannot query transactions if a currency doesn't have a subscanBaseUrl
+    */
+    if (this.networkInfo.subscanBaseUrl === '') {
```

Comment:
Optional: Could we make `subscanBaseUrl: string | undefined`? I really don't like blank-string checks, if we can use undefined instead.